quickstarts/Tuning.ipynb

```diff
@@ -157,7 +157,7 @@
       },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    base_model = base_model = [m for m in tunable_models if 'flash' in m.name][-1]  # models/gemini-1.5-flash-001-tuning
</pre><p>Looks like a typo here: extra <code>base_model =</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/google-gemini/cookbook/pull/262/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
Fixed.